include/clang/Interpreter/CppInterOp.h

```diff
@@ -550,6 +557,13 @@ namespace Cpp {
   CPPINTEROP_API TCppFunction_t
   InstantiateTemplateFunctionFromString(const char* function_template);
 
+  // Find best template match based on explicit template parameters and arg
+  // types
```

Comment:
```suggestion
  /// Finds best template match based on explicit template parameters and argument
  /// types
```

We need documentation on the parameters.